### DISCUSSION TITLE
React native integration with Appium

### DISCUSSION THREAD

**Discussion by ghost** [appium/appium#15589] [Q&A]
Hi Team ,

     
I am working for a company so they want e2e testing with Appium but there are no docs or resources on it on Website. So , 
 could you please help me through it or provide any docs or resources on it.

**Comment by KazuCocoa** [comment]
What kind of documentation do you expect?
In general, Appium does black-box testing. So the fundamental docs are each driver information and command stuff. 

e.g.
https://appium.io/docs/en/drivers/ios-xcuitest/
https://appium.io/docs/en/drivers/android-uiautomator2/
https://appium.io/docs/en/writing-running-appium/finding-elements/

Accessibility feature generally helps automation, so like https://reactnative.dev/docs/accessibility and https://reactnative.dev/docs/testing-overview also may help. It is not Appium specific thing.

You also can find issues related to react native. Appium basically access to each element via OS' feature, so in this case you will need to find workarounds in react native layer. (It seems it depends on your situation...)
https://github.com/appium/appium/search?q=react+native&type=issues

https://discuss.appium.io/ will help you to ask some general questions.
